lib/main_containers.pm

```diff
@@ -107,7 +107,7 @@ sub load_host_tests_podman {
     loadtest 'containers/podman_bci_systemd';
     loadtest 'containers/podman_pods';
     # Default for ALP is Netavark
-    loadtest('containers/podman_network_cni') unless (is_alp || is_sle_micro('6.0+'));
+    loadtest('containers/podman_network_cni') unless (is_alp || is_sle_micro('5.5+'));
```

Comment:
I thought slem 5.5 is a specific case only for PC as the image is newer than non PC slem 5.5

Reply:
Yes, probably I should add a check around `...is_publiccloud` too, but in what case?
_exclude_ cni for something like (is_pubc and is_slemicro=5.5) ?

Reply:
yes

Reply:
done